<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2000</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { 
background-color: #FFFFFF ;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="bas1" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="bas2" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>110</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>GROUND STATION SOFTWARE</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="temp" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>570</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>301</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>58</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
background-color: black;
}</string>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height2" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>340</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QCustomPlot" name="lspeed" native="true">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>570</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>710</y>
      <width>210</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>30</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #5D6D7E;
}</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCustomPlot" name="height1" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>340</y>
      <width>210</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget">
    <property name="geometry">
     <rect>
      <x>969</x>
      <y>-1</y>
      <width>700</width>
      <height>430</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans</family>
      <pointsize>30</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: #5D6D7E;
}</string>
    </property>
    <property name="text">
     <string>Next Video</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QQuickWidget" name="quickWidget_2">
    <property name="geometry">
     <rect>
      <x>968</x>
      <y>450</y>
      <width>701</width>
      <height>391</height>
     </rect>
    </property>
    <property name="resizeMode">
     <enum>QQuickWidget::SizeRootObjectToView</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQuickWidget</class>
   <extends>QWidget</extends>
   <header location="global">QtQuickWidgets/QQuickWidget</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
